### PR TITLE
Matthew Murray

### PR DESCRIPTION
**Pull request by matthewMurray14**
# Checkpoint Rubric

This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

## Checkpoint 1: Company/Organization Website following Wireframe
- [x] A CSS file is effectively linked to the HTML file in the Head section: 10 points
- [ ] A link to a CSS reset file is included in the Head section of the HTML file: 10 points 
- [ ] The webpage links to another page (multi-page website): 10 points
- [x] The company/orgnanization name is lined up with the navigation menu at the top of the page: 10 points
- [ ] The title is centered in front of a background image: 10 points
- [ ] The body images are sized and formatted according to the wire frame: 10 points
- [ ] The body text is sized, positioned, and formatted to match the fire frame: 10 points
- [x] Flexbox or floats are used to accurately replicate the layout of the wireframe: 10 points
- [ ] The footer consists of a background image, centered text, and 2 buttons that function as links: 10 points
- [ ] The student demonstrated mastery of HTML/CSS by building a website of professional caliber: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 2: Foundry Winery Website
- [ ] Responsive navbar: 20 points
- [ ] Responsive site (resizes to fit desktop, tablet, and mobile views): 30 points 
- [ ] Matched layout of original site: 20 points
- [ ] Matched font of original site: 10 points
- [ ] Properly commented code: 10 points
- [ ] Properly indented code: 10 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 3: Learning Team Project
- [ ] Mockup meets customer requirements: 25 points
- [ ] Website meets customer requirements: 25 points
- [ ] Learning team worked together effectively and contributed equally: 25 points
- [ ] Class presentation was engaging and effective: 25 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

## Checkpoint 4: Personal Portfolio Website
- [ ] Responive: 20 points
- [ ] Evidence of Effective Use of Design Principles: 20 points
- [ ] Links to Resume: 20 points
- [ ] Links to at least 3 Projects: 20 points
- [ ] Effective Communicates Desired Message ("Hire me"): 20 points

Specific Comments:

*Comments may also be made on the file itself using github's commenting tools.*

### CODE REVIEW

File: 02Lesson/01Homework/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
           and deadlines under strict time constraints. In addition to my strong commitment
           and desire to succeed, I am able to work independently or as a team.</p>
    <h4>Experience</h4>
-        <p>I am a self taught and gradguate of the Austin Coding Academy
+        <p>I am a self taught and gradguate of the Austin Coding Academy</p>
```
Comment (genarmonroe) [inline 02Lesson/01Homework/index.html]: Hi Matt- a lot of your code looks good so far as I can see (as a noob)! My only recommendation is to consider spellcheck of content for projects you do plan on using for your future portfolio (for instance here- "graduate").

File: 02Lesson/01Homework/index.html [hunk 2]
```diff
@@ -57,5 +57,6 @@
         </table>
         <div id="footer">
             <p>2005 Willow Creek Dr, Austin, Tx 78741 | tel: (512)-838-8149</p>
-        </div></body>
+        </div>
```
Comment (arxrz08) [inline 02Lesson/01Homework/index.html]: good job @matthewMurray14

Comment (arxrz08) [reply 02Lesson/01Homework/index.html]: Were you able to find out what was missing?

**Review by arxrz08 — APPROVED** [review]

File: .ARCHIVE/02Lesson/01Homework/index.html [hunk 3]
```diff
@@ -1,8 +1,63 @@
...
+<!-- this resume is ment to be updated regularly for profesional use -->
+
 <head>
-  <title></title>
+    <link type="text/css" rel="stylesheet" href="stylesheet.css" />
+    <title>Matthew Murray - Resume</title>
 </head>
+
 <body>
-</body>
```
Comment (stefanbemelmans) [inline .ARCHIVE/02Lesson/01Homework/index.html:7]: extra </body> tag

**Pull request closed without merging**